src/arithmetic/arithmetic_expression.c

```
@@ -207,17 +223,22 @@ AR_ExpNode *AR_EXP_NewOpNode
 	return node;
 }
 
+static AR_EXP_Result _AR_EXP_EvaluateBorrowRecord(AR_ExpNode *node, const Record r,
+	SIValue *result);
+
 static inline AR_ExpNode *_AR_EXP_InitializeOperand(AR_OperandNodeType type) {
 	AR_ExpNode *node = rm_calloc(1, sizeof(AR_ExpNode));
```

Cppcheck (2.10-2)

[error] Common realloc mistake

(memleakOnRealloc)